dev/breeze/src/airflow_breeze/shell/shell_builder.py

```diff
@@ -206,7 +205,7 @@ def compose_files(self):
         )
         # mssql based check have to be added
         if self.backend == 'mssql':
-            docker_filesystem = get_filesystem_type()
+            docker_filesystem = get_filesystem_type('.')
```

Comment:
There was a filesystem missing here. I think "." will be fine, but somethign to check on CI.